### ISSUE TITLE
[bug]: Radio inputs in documentation don't have a label

### ISSUE THREAD

**Issue by leagrdv** [swisspost/design-system#5251]
### 🐞 Describe the bug

The radio inputs in [Guidelines > Form labels](https://next.design-system.post.ch/?path=/docs/13fb5dfe-6c96-4246-aa6a-6df9569f143f--docs) is not accessible as their labels are not within a `<label>` tag.

### 📷 Screenshots

![Image](https://github.com/user-attachments/assets/8fa88bc5-2d70-4c70-ae1c-609176abf1f4)
